tools/src/main/java/com/nedap/archie/archetypevalidator/ValidationResult.java

```diff
@@ -124,7 +124,12 @@ public String toString() {
         for(ValidationMessage message:errors) {
             result.append(message);
             result.append("\n");
-
+        }
+        if(overlayValidations != null) {
+            for(ValidationResult overlayValidation:overlayValidations) {
+                result.append("\n");
+                result.append(overlayValidation);
+            }
```

Comment:
Should the `overlayValidations` also be checked in the `passes` and/or `hasWarningsOrErrors` methods?

Reply:
Not needed. When an overlay validation is added that causes it to fail, the main validation gets an error about the overlay failing.